### DISCUSSION TITLE
Cannot access commit previews

### DISCUSSION THREAD

**Discussion by makhlwf** [community/community#128850] [Other Feature Feedback, Questions, & Ideas]
Cannot access the code lines in the commit previews

**Comment by m1her** [comment]
Check [This](https://www.freecodecamp.org/news/git-reverting-to-previous-commit-how-to-revert-to-last-commit/)

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @makhlwf ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
